script/test

```diff
@@ -7,4 +7,4 @@ set -e
 #fi
 
 echo "Running tests..."
-docker-compose run --service-ports -e --rm web bash -c "cd fixtures && ./copy_media.sh && cd .. && python manage.py test cms"
+docker-compose run --service-ports -e --rm web bash -c "cd fixtures && ./copy_media.sh && cd .. && python manage.py test cms $*"
```

Comment:
This doesn't feel related to the PR, should it be in its own, or at least its own commit?

Reply:
Done.